### PR TITLE
URL changes

### PR DESCRIPTION
**Pull request by callumacrae**
🐉🤠🐉🤠🐉🤠🐉🤠🐉🤠🐉🤠

(work in progress)

### CODE REVIEW

File: app/controllers/core_induction_programmes/mentor_materials_controller.rb [hunk 1]
```diff
@@ -1,11 +1,13 @@
 # frozen_string_literal: true
 
-class CoreInductionProgrammes::MentorMaterialsController < ApplicationController
+class CoreInductionProgrammes::MentorMaterialsController < CoreInductionProgrammes::LessonsController
```
Comment (kicferk1) [inline app/controllers/core_induction_programmes/mentor_materials_controller.rb:3]: Are we doing all of this controller inheritance to have access to `load_xxx` methods that are shared between them?

If so, I would really prefer those load methods to be in a helper, and include that helper in all relevant controllers. That saves us the need to worry about what actions we need to skip, and what needless methods we inherit.

File: db/migrate/20210624134755_add_slug_to_cips.rb [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+SLUGS = {
+  "Ambition Institute": "ambition",
+  "Education Development Trust": "edt",
+}.freeze
```
Comment (kicferk1) [inline db/migrate/20210624134755_add_slug_to_cips.rb:6]: Probably need more

File: db/schema.rb [hunk 3]
```diff
@@ -42,6 +42,7 @@
     t.datetime "updated_at", precision: 6, null: false
     t.uuid "course_year_one_id"
     t.uuid "course_year_two_id"
+    t.string "slug"
```
Comment (kicferk1) [inline db/schema.rb:45]: I would expect this to cause some changes in the seed dump, but not 400 lines. Before we merge it, could you make sure you get latest data from develop (on old db version), migrate it to add the slug and run `rails cip_seed_dump` afterwards? I hope for like 10 line diff (just for core induction programmes) :)

File: db/migrate/20210624134755_add_slug_to_cips.rb (hunk 2 above)
Comment (callumacrae) [reply db/migrate/20210624134755_add_slug_to_cips.rb:6]: UCL -> ucl
Teach First -> teach-first

are both basically what we want already so no need to override them

File: db/schema.rb (hunk 3 above)
Comment (callumacrae) [reply db/schema.rb:45]: yea i'm pretty sure most of the diff is the changes you made last week but backwards - i'll update from develop at some point :)

**Comment by callumacrae** [comment]
note to self: i have somehow absolutely destroyed the seed dump file

**Pull request closed without merging**